src/components/Card.js

```diff
@@ -15,26 +25,49 @@ class Card {
     return cardElement;
   }
 
-  _handleLike = () => {
+  isLiked() {
+    return this._isLiked;
+  }
+
+  handleLike = () => {
     this._likeButton.classList.toggle("gallery__card-like_active");
   };
 
-  _handleDelete = () => {
+  handleDelete = () => {
     this._element.remove();
   };
 
+  updateLikeView() {
+    // write a public method to update like icon after successful response
+    // it just needs to toggle a class probably
+    // this._likeButton.classList.toggle("gallery__card-like_active");
+    if (this._isLiked) {
+      this._likeButton.classList.add("gallery__card-like_active");
+    } else {
+      this._likeButton.classList.remove("gallery__card-like_active");
+    }
+  }
+
   _setEventListeners() {
-    this._likeButton.addEventListener("click", this._handleLike);
-    this._cardTrash.addEventListener("click", this._handleDelete);
+    this._deleteFormButton.addEventListener("submit", this._handleDelete);
```

Comment:
`    this._deleteFormButton.addEventListener("submit", this._handleDelete);
`

Please, delete it

Every submit handling should be done inside `PopupWithForm` or `PopupWithDelete`